### ISSUE TITLE
Detect detox!

### ISSUE THREAD

**Issue by GantMan** [infinitered/solidarity-react-native#7]
If a user has Detox cli, go ahead and add detox to the snapshot.  See code for many examples of exactly this.

**Comment by GantMan** [comment]
```
  "Detox": [
    {
      "rule": "cli",
      "binary": "detox"
    },
    {
      "rule": "cli",
      "binary": "applesimutils"
    }
  ],
```

**Comment by AugustasV** [comment]
I will try

**Comment by AugustasV** [comment]
Anyone can try. Too hard for me.

**Comment by GantMan** [comment]
If you'd like help @AugustasV - hit me up on http://community.infinite.red and I'll help.  Otherwise thanks for checking it out!

**Comment by codeithuman** [comment]
I'm going to take a crack at it.

**Comment by GantMan** [comment]
closed by #8
